project/src/main/webapp/index.html

```diff
@@ -8,7 +8,14 @@
   </head>
   <body>
     <div class="section">
-      <div id="hamburger-icon"><div></div></div>
+      <div id="hamburger-icon" onclick="toggleNavBar()"><div></div></div>     
+      <nav id="nav" style="display: none">
```

Comment:
nit: can the style be moved to the css file?

Reply:
the way the JS is written right now, without the style, the navbar will only open on the second click of the nav icon. Do you have a recommendation on how to do it without needing the style?

Reply:
can you put 'display:none' as a characteristic of the 'nav' id in the css? like #nav { display: none }

Reply:
Since the JS is accessing the HTML element, that doesn't work either because without the style on the tag, it's technically not specified initially, which is why it only opens on the second click (clicking the first time and thereafter assigns and reassigns it). Also, the nav tag actually already has display: none in the css, but I'm guessing that's redundant, so I can remove that.